### PR TITLE
Rearrange code and make good structure, Fix Banner Ads size and position

### PR DESCRIPTION
**Pull request by rohitjakhar**
I make new package and rearrange whole code that are working well 
Fix Banner Ads Size so it show in better manner.

### CODE REVIEW

File: .idea/compiler.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="CompilerConfiguration">
+    <bytecodeTargetLevel target="1.8" />
+  </component>
+</project>
```
Comment (DeekshithRajBasa) [inline .idea/compiler.xml:6]: add new line

File: .idea/jarRepositories.xml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+      <option name="name" value="$USER_HOME$/Android/Sdk/extras/google/m2repository" />
+      <option name="url" value="file:$USER_HOME$/Android/Sdk/extras/google/m2repository" />
+    </remote-repository>
+    <remote-repository>
+      <option name="id" value="$USER_HOME$/Android/Sdk/extras/m2repository" />
+      <option name="name" value="$USER_HOME$/Android/Sdk/extras/m2repository" />
+      <option name="url" value="file:$USER_HOME$/Android/Sdk/extras/m2repository" />
+    </remote-repository>
+  </component>
+</project>
```
Comment (DeekshithRajBasa) [inline .idea/jarRepositories.xml:40]: add a new line to bottom

File: .idea/vcs.xml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="VcsDirectoryMappings">
+    <mapping directory="" vcs="Git" />
+  </component>
+</project>
```
Comment (DeekshithRajBasa) [inline .idea/vcs.xml:6]: add new line

File: .idea/misc.xml [hunk 4]
```diff
@@ -25,7 +25,7 @@
       </value>
     </option>
   </component>
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_7" project-jdk-name="JDK" project-jdk-type="JavaSDK">
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_8" default="false" project-jdk-name="1.8" project-jdk-type="JavaSDK">
```
Comment (DeekshithRajBasa) [inline .idea/misc.xml:28]: any specific reason for 1.8?

File: app/src/main/res/layout/content_main.xml [hunk 5]
```diff
@@ -22,26 +22,24 @@
...
-        ads:layout_constraintStart_toStartOf="parent"
-        ads:layout_constraintTop_toTopOf="parent"></com.google.android.gms.ads.AdView>
+        app:adSize="BANNER"
+        app:adUnitId="ca-app-pub-7250174976428336/1382229664"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent" />
 
 
 </android.support.constraint.ConstraintLayout>
```
Comment (DeekshithRajBasa) [inline app/src/main/res/layout/content_main.xml:45]: add a new line to the bottom

**Review by DeekshithRajBasa — CHANGES_REQUESTED** [review]
Please make the necessary changes

File: app/src/main/res/layout/content_main.xml (hunk 5 above)
Comment (DeekshithRajBasa) [reply app/src/main/res/layout/content_main.xml:45]: Please resolve the conflicts

**Comment by DeekshithRajBasa** [comment]
@rohitjakhar Planning to close this PR as it is been open for a while now, can you please make those requested changes and resolve the conflicts? I'll merge it once you did the changes.

File: .idea/misc.xml (hunk 4 above)
Comment (rohitjakhar) [reply .idea/misc.xml:28]: Yes, Java 8 lot of new features.

File: app/src/main/res/layout/content_main.xml (hunk 5 above)
Comment (rohitjakhar) [reply app/src/main/res/layout/content_main.xml:45]: Where is conflit ?

Comment (DeekshithRajBasa) [reply app/src/main/res/layout/content_main.xml:45]: could you pls see the bottom of the PR and resolve the conflicts 
image

**Pull request closed without merging**